DataFormats/HeavyIonEvent/interface/HFFilterStruct.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DataFormats_HeavyIonEvent_HFFilterStruct_H
+#define DataFormats_HeavyIonEvent_HFFilterStruct_H
+
+namespace pat {
+  struct HFFilterStruct {
```

Comment:
I don't think "Struct" suffix is useful, I can't recall of other examples in DataFormats.

Reply:
Is better to use HFFilter only?

Reply:
HFFilterInfo or HFFilterSummary?

Also, does this have to be in the pat namespace? The code is not explicitly in the PhysicsTools/ or DataFormats/PatCandidates

Reply:
ok, I will use HFFilterInfo. 

Actually does not need to be pat namespace, I can use reco namespace. However,
the code is called by PhysicsTools/PatAlgos/python/slimming/slimming_cff.py in order to run on PAT.

Reply:
Do you prefer pat, reco, or maybe another/new namespace?

Reply:
I would say `reco`, as the other classes in DataFormats/HeavyIonEvent that are stored in the event seem to be in that namespace: https://github.com/cms-sw/cmssw/blob/master/DataFormats/HeavyIonEvent/src/classes_def.xml

Reply:
looking at the current use patterns, I'd say reco or none

Reply:
I move to redo in commit 78b0a76436417538aefafc8a492dd0314831b794